src/components/sbb-card/sbb-card.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, h, Host, JSX, Prop, State } from '@stencil/core';
+import { InterfaceSbbCardAttributes } from './sbb-card.custom';
+
+/**
+ * @slot unnamed - Slot to render the content.
+ * @slot badge - Slot to render `<sbb-card-badge>`.
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-card.scss',
+  tag: 'sbb-card',
+})
+export class SbbCard {
+  /**
+   * Size variant, either xs, s, m, l, xl and xxl.
+   */
+  @Prop() public size?: InterfaceSbbCardAttributes['size'] = 'm';
+
+  /* @internal */
+  @State() private _hasBadge = false;
+
+  /**
+   * It is used internally to show the `<sbb-card>`.
+   *
+   * @param size The size selected.
+   * @returns True whether size is equal to m, l, xl or xxl.
+   */
+  private _showSBBBadge(size: string): boolean {
```

Comment:
don't pass size as it is a class field, you can access with `this.size`
```suggestion
  private _showSbbBadge(): boolean {
```